code/game/objects/items/stacks/tape.dm

```diff
@@ -19,29 +19,43 @@
 	var/list/conferred_embed = EMBED_HARMLESS
 	var/overwrite_existing = FALSE
 
-/obj/item/stack/sticky_tape/afterattack(obj/item/I, mob/living/user)
+/obj/item/stack/sticky_tape/afterattack(obj/item/I, mob/living/user, proximity)
+	if(!proximity)
+		return
+
 	if(!istype(I))
 		return
 
+	if(I.embedding)
```

Comment:
Redundant check per the existing one below it, and being able to replace one type of embed for another is intended behavior, like using pointy tape or super sticky tape to replace normal sticky tape

Reply:
If it's intended it's kind of borked as hell and doesn't really do much? You can't unsticky tape an item, so do we really need the ability to worsen an item?

On the topic of taping a taped item, you have surely seen the chat spamming that can result from someone taping an item multiple times, right? The name stacking doesn't stop.

(Also maybe I'm a dumb cat but if I put this check below the next, it never seemed to get checked at all. I know right now that second check isn't being reached so I'll try and fix that.)

Reply:
You can apply better sticky tape to it as well, or maybe someone wants to make purposely defang'd shurikens, either way it's intended functionality. As for the name spam, check to see if the item already has the tape's prefix in it, and if it does, don't apply the prefix again

Reply:
I don't agree that this is the way to do it. If you intended for tape to be replaceable it should be done in some way to check that it is taped, clear the previous statistics and everything associated with the tape, and with some method of removing it without first adding more tape. This all seems like a hot mess since none of this is discernible from a player point of view what minor statistical alterations have taken place and how much of an impact this has had on a weapons stats. Even the pointy tape may not be an improvement over what is currently on the item.

I also can't be arsed to do that right now and I don't think anything of significance is lost by preventing this.